src/Essentials/src/Vibration/Vibration.android.cs

```diff
@@ -45,6 +45,8 @@ void PlatformVibrate(TimeSpan duration)
 			Permissions.EnsureDeclared<Permissions.Vibrate>();
 
 			var time = (long)duration.TotalMilliseconds;
+			if (time == 0)
```

Comment:
```suggestion
			if (time <= 0)
```

While we do clamp the duration at the shared code level, I think it's a good idea to make this a bit more resilient here in case anyone ever copies and pastes or reuses this code somewhere else without considering the value first.